### ISSUE TITLE
Fuel Breaks in ELM Fire

### ISSUE THREAD

**Issue by johnsonas6** [lautenberger/elmfire#4]
Hi Chris,

I'm interested in modeling fuel break effectiveness in ELM fire with a single ignition and with random ignitions. Most importantly I am curious about advice on using manipulated landscapes (not from cloudfire) and different weather data.

Thank you,

Andrew

**Comment by lautenberger** [comment]
Hi Andrew,

I'll set up another tutorial case when I get a chance but until then:

* Weather - [Tutorial 2](https://elmfire.io/tutorials/tutorial_02.html) provides an example of how you can create the wind speed/direction and fuel moisture rasters from a .csv file. 

* Random ignitions - The  [Monte Carlo Analysis](https://elmfire.io/user_guide/monte_carlo.html) section of the User Guide shows how you can set up randomized ignition locations

* Updating fuel inputs for fire breaks - [gdal_rasterize](https://gdal.org/programs/gdal_rasterize.html) is a good tool for this. You can burn vector features (lines or polygons) into the fuel input rasters to reflect fuel treatments or fire breaks, for example as a change in fuel model or reduction in canopy bulk density, etc.

**Comment by johnsonas6** [comment]
Hi Chris,

Thanks for your help! I'm fairly familiar with how to manipulate LANDFIRE
data in R or Arc, but what I'm not sure of is how to get my new landscapes
into ELM Fire. I could be misunderstanding something, but it seems that it
pulls the raster data from some online repository?

Thanks,

Andrew

**Comment by lautenberger** [comment]
Hey Andrew-

Fuel inputs are read in from a series of GeoTiff files for surface fuel model, canopy base height, etc. - basically all of the layers in a FARSITE .lcp file are read in from individual GeoTiffs. The `fuel_wx_ign.py` microservice provides an easy way to grab stock LANDFIRE inputs by version, but you can also provide your own inputs if you'd like. 

In [Tutorial 3](https://elmfire.io/tutorials/tutorial_03.html), for example, `fuel_wx_ign.py` is used to download LANDFIRE 2.2.0 fuel & topo inputs and store them locally. They land in the `inputs` folder for later use by ELMFIRE. To run ELMFIRE with modified fuels you could copy the `inputs` folder to say `inputs_modified` and then burn features to `fbfm40.tif` to represent fuel breaks or fuel treatments. To run ELMFIRE with these modified fuels you'd then edit `elmfire.data` to tell ELMFIRE where the modified fuels are by changing the line `FUELS_AND_TOPOGRAPHY_DIRECTORY = './inputs'` to `FUELS_AND_TOPOGRAPHY_DIRECTORY = './inputs_modified'`.

**Comment by johnsonas6** [comment]
Hi Chris,

Apologies for the delayed response, I've been out in the field helping
collect some pre-salvage logging forest inventory data. This is incredibly
helpful, thank you. I was under the impression that these files were being
pulled from some specially hosted database of LANDFIRE data, but this makes
more sense.

I'm going to be in Portugal for a conference/collaboration trip over the
next couple of weeks and then am heading straight into the fire season.
I'll try to work on this periodically over days off over the summer, but
will probably be mostly taking a break from science until the fall.
Basically, if I run into any hiccups and need to reach out, it might not be
for a while and I hope this is okay. Thanks again!

Warmly,

Andrew

**Comment by lautenberger** [comment]
Closing this issue since it's almost 2 years old but please submit a new issue if you're still working in this area and encountering problems.
